cmd/atlas/atlas.go

```diff
@@ -36,6 +36,10 @@ import (
 func Execute() {
 	ctx := telemetry.NewContext()
 	rootCmd := atlas.Builder()
+	// append here to avoid a recursive link on generated docs
+	rootCmd.Long += `
+
+For more information, refer to the docs: https://www.mongodb.com/docs/atlas/cli/stable/`
```

Comment:
This is the language we use in docs, feel free to take or leave the suggestion:
```suggestion
To learn more, see our documentation: https://www.mongodb.com/docs/atlas/cli/stable/`
```